iothub/device/src/IPayloadConvention.cs

```diff
@@ -4,25 +4,30 @@
 namespace Microsoft.Azure.Devices.Client
 {
     /// <summary>
-    ///
+    /// The payload convention class.
     /// </summary>
+    /// <remarks>The payload convention is used to define a specific serializer as well as a specific content encoding. For example, IoT has a <see href="https://docs.microsoft.com/en-us/azure/iot-pnp/concepts-convention">convention</see> that is designed to make it easier to get started with products that use specific conventions by default.</remarks>
     public abstract class IPayloadConvention
     {
         /// <summary>
-        ///
+        /// The serializer used for the payload of the <see cref="Message"/>.
```

Comment:
we shouldn't reference message here, for purposes of pnp Message is now an internal implementation detail